sql/step_78_logout_menu_item.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO OH_MENUITEM VALUES ('logout','angal.menu.btn.logout','angal.menu.logout','x','L','main','none','N',13);

```

Comment:
This file needs to be renamed: `step_82_logout_menu_item.sql`.  And make the corresponding change in the `step_04_all_following_steps.sql` file above too.